public/less/themes/arc-green.less

```diff
@@ -85,6 +85,59 @@ body {
     color: #9e9e9e;
 }
 
+/* firefox scroll bars */
+
+html,
+body,
+* {
+    scrollbar-width: thin;
+    scrollbar-color: #87ab63 #ffffff;
```

Comment:
This seems to display a white background for the slider on firefox.
```suggestion
    scrollbar-color: #87ab63 #000;
```

Reply:
* In firefox the `background for the slider` currently could not be set. (iirc this could only be done by system style theme)
* second parameter is for rounded borders on scroll bar only. 
Here is a screenshot, second parameter set to red, Zoom 270%
![image](https://user-images.githubusercontent.com/24939127/59957473-fce02a80-9498-11e9-99bd-7b17ae2e15be.png)

Reply:
But i will add your changes:
![image](https://user-images.githubusercontent.com/24939127/59957548-9f001280-9499-11e9-8e5f-1f8eb0f8f70b.png)

Reply:
I will revalidate but on my setup it changed the background (FF on linux).

Reply:
Re-tested on FF 67.0b7 on windows
![image](https://user-images.githubusercontent.com/4052400/60163963-fd602480-97fc-11e9-8962-746f40985e67.png)

Reply:
my FF (67.0.3 (64-Bit)) screenshots where taken on ubuntu/plasma5. May may system theme breaks thinks, as my scrollbar is completely white :)